### DISCUSSION TITLE
Disable auto-capitalize on mac

### DISCUSSION THREAD

**Discussion by colinlienard** [tauri-apps/tauri#7635] [Q&A] [answered]
Hi!

This isn't really a Tauri issue but I haven't found the solution to my problem anywhere else so I'm asking here

I'm working on [GitLight](https://github.com/colinlienard/gitlight) and I have an input field wher users can enter a GitHub organization name followed by a personal access token to enable the use of notifications on private repositories.

The organization name must be written without spelling mistakes or it wouldn't work. But when de-focusing the input, the case is automatically changed and users don't always see it, leading to misunderstandings.

![image](https://github.com/tauri-apps/tauri/assets/64312634/94a25e47-aab0-4376-a00b-0327867b742d) ![image](https://github.com/tauri-apps/tauri/assets/64312634/c70fac31-5c0b-460e-8329-b134e6d4af64)

- This behaviour doesn't occur in the web version, only in the desktop app.
- I have already tried setting `autocapitalize="off"` on the input with no success.
- I know the user can disable this behaviour in the os settings, but I would like it to be disabled specifically for my app, is that possible?

**Comment by reyamir** [comment, marked as answer]
please try this, it work in my case
```
spellCheck={false}
autoComplete="off"
autoCorrect="off"
autoCapitalize="off"
```

**Reply by colinlienard** [reply]
Disabling `spellcheck` and `autocorrect` works, thanks a lot!
